src/components/FreeType/FreeType.scss

```diff
@@ -27,10 +27,37 @@
     color: $color-blue-navy;
     background: $color-white;
     border: 0.063rem solid $color-grey-light;
-    box-shadow: 0rem 0.063rem 0.125rem $color-blue-dark;
+    box-shadow: 0rem 1px 2px $color-blue-dark;
```

Comment:
line 30 (and anywhere else it is present) please change pixels back to rems, everythng must be rem! :)